### PR TITLE
Add test suite

### PR DESCRIPTION
**Pull request by rubensworks**
This PR adds 29 streaming-specific toRdf tests.

7 of these tests are negative evaluation tests where the required key order is not followed.

The other tests check the essential key orders where streaming parsers could potentially break.
(There are _much_ more of these tests possible, but these ones should already cover the basics)

I've named the manifest for these tests "stream-toRdf", so that other streaming manifests could be added in the future as well, such as "stream-fromRdf" (I don't foresee myself having time to create this any time soon).

***
<a href="https://pr-preview.s3.amazonaws.com/rubensworks/json-ld-streaming/pull/5.html" title="Last updated on Mar 24, 2020, 4:05 PM UTC (1bf2f5e)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/json-ld-streaming/5/77f762b...rubensworks:1bf2f5e.html" title="Last updated on Mar 24, 2020, 4:05 PM UTC (1bf2f5e)">Diff</a>

### CODE REVIEW

File: tests/README.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+* [stream-toRdf](stream-toRdf-manifest.jsonld) tests have _input_ and _expected_ documents.
+  The _expected_ results can be compared using [RDF Dataset Isomorphism](https://www.w3.org/TR/rdf11-concepts/#dfn-dataset-isomorphism).
+
+  Stream-ToRdf tests may have a `expandContext` option, which is treated
+  as an IRI relative to the manifest.
+
+  For *NegativeEvaluationTests*, the result is a string associated with the expected error code.
+
+Unless `processingMode` is set explicitly in a test entry, `processingMode` is compatible with both `json-ld-1.0` and `json-ld-1.1`.
+
```
Comment (gkellogg) [inline tests/README.md]: processingMode probably isn't relevant, as there is no 1.0 mode for streaming processors.

File: tests/README.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+Test results that include a context input presume that the context is provided locally, and not from the referenced location, thus the results will include the content of the context file, rather than a reference.
+
+## JSON-LD Object comparison
+
+If algorithms are invoked with the `ordered` flag set to `true`, simple JSON Object comparison may be used, as the order of all arrays will be preserved (except for _fromRdf_, unless the input quads are also ordered). If `ordered` is `false`, then the following algorithm will ensure arrays other than values of `@list` are compared without regard to order.
+
```
Comment (gkellogg) [inline tests/README.md]: You may want to remove parts of this that aren't relevant to the tests. Easy enough to add back in of you do a streaming-fromRdf spec.

File: tests/stream-toRdf-manifest.jsonld [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+  "baseIri": "https://w3c.github.io/json-ld-streaming/tests/",
+  "sequence": [
+    {
+      "@id": "#te001",
+      "@type": ["jld:NegativeEvaluationTest", "jld:ToRDFTest"],
+      "name": "@context must come before @id",
+      "purpose": "Generate an error if @context comes after @id",
+      "input": "stream-toRdf/e001-in.jsonld",
+      "expectErrorCode": "invalid streaming key order",
+      "option": {"specVersion": "json-ld-1.1"}
```
Comment (gkellogg) [inline tests/stream-toRdf-manifest.jsonld]: specVersion isn't really relevant.

**Comment by rubensworks** [comment]
Thanks for the feedback!

**Pull request merged**